R/intervals.R

```diff
@@ -0,0 +1,625 @@
+#' @title Generate columns to help describe numeric variable intervals
```

Comment:
This is another long file that might benefit from being broken into smaller files.